inference-engine/tests/functional/plugin/cpu/single_layer_tests/roialign.cpp

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "functional_test_utils/layer_test_utils.hpp"
+
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+
+namespace CPULayerTestsDefinitions {
+namespace {
+    int pooledH;
+    int pooledW;
+    float spatialScale;
+    int samplingRatio;
+    std::vector<float> proposalVector;
+    std::vector<size_t> roiIdxVector;
+    std::string mode;
+    std::vector<size_t> inputShape;
+}  // namespace
+
+typedef std::tuple<
+        int,                  // bin's column count
+        int,                  // bin's row count
+        float,                // scale for given region considering actual input size
+        int,                  // pooling ratio
+        std::vector<float>,   // coordinate vector: left_top_x, left_top_y, right_bottom_x, right_bottom_y
+        std::vector<size_t>,  // batch id's vector
+        std::string,          // pooling mode
+        std::vector<size_t>   // feature map shape
+> ROIAlignSpecificParams;
+
+typedef std::tuple<
+        ROIAlignSpecificParams,
+        InferenceEngine::Precision,     // Net precision
+        LayerTestsUtils::TargetDevice   // Device name
+> ROIAlignLayerTestParams;
+
+typedef std::tuple<
+        CPULayerTestsDefinitions::ROIAlignLayerTestParams,
+        CPUSpecificParams> ROIAlignLayerCPUTestParamsSet;
+
+class ROIAlignLayerCPUTest : public testing::WithParamInterface<ROIAlignLayerCPUTestParamsSet>,
+                             virtual public LayerTestsUtils::LayerTestsCommon, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<ROIAlignLayerCPUTestParamsSet> obj) {
+        CPULayerTestsDefinitions::ROIAlignLayerTestParams basicParamsSet;
+        CPUSpecificParams cpuParams;
+        std::tie(basicParamsSet, cpuParams) = obj.param;
+        std::string td;
+        Precision netPr;
+        ROIAlignSpecificParams roiPar;
+        std::tie(roiPar, netPr, td) = basicParamsSet;
+        std::tie(pooledH, pooledW, spatialScale, samplingRatio,
+                 proposalVector, roiIdxVector, mode, inputShape) = roiPar;
+        std::ostringstream result;
+        result << "ROIAlignTest_";
```

Comment:
may i ask you modify test name creation for more detail test case name description

Reply:
Done